### PR TITLE
Add the update control variables to the demo configuration

### PR DESCRIPTION
**Pull request by oleorhagen**
This adds the possibility to configure the udpate control configuration
variables in 'mender.conf'.

This is done through adding:

* MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS
* MENDER_UPDATE_CONTROL_MAP_BOOT_EXPIRATION_TIME_SECONDS

variables, which, if set, are added to the rootfs local configuration file in
/etc/mender/mender.conf.

It also adds a default value for these variables in the demo layer, in which the
values are set to 90 and 45, respectively.

Not documenting it in purpose, as we favor the method of adding a custom
mender.conf file to the build instead of setting up even more MENDER_
variables.

Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>
Signed-off-by: Lluis Campos <lluis.campos@northern.tech>

### CODE REVIEW

File: meta-mender-demo/recipes-mender/mender-client/mender-client_%.bbappend [hunk 1]
```diff
@@ -2,6 +2,9 @@ MENDER_UPDATE_POLL_INTERVAL_SECONDS_mender-client-install = "5"
 MENDER_INVENTORY_POLL_INTERVAL_SECONDS_mender-client-install = "5"
 MENDER_RETRY_POLL_INTERVAL_SECONDS_mender-client-install = "30"
 
+MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS = "90"
+MENDER_UPDATE_CONTROL_MAP_BOOT_EXPIRATION_TIME_SECONDS = "45"
```
Comment (lluiscampos) [inline meta-mender-demo/recipes-mender/mender-client/mender-client_%.bbappend:6]:
```suggestion
MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS_mender-client-install = "90"
MENDER_UPDATE_CONTROL_MAP_BOOT_EXPIRATION_TIME_SECONDS_mender-client-install = "45"
```

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Thanks for taking this.

Two remarks:

* Should we align the values with https://github.com/mendersoftware/meta-mender/pull/1416/files ?
* Good changelog message, but we should slightly extend it (or create a new one?) to say that we are setting them to 90/45 (or whatever we decide on) for `meta-mender-demo`.

See comment also.

**Comment by oleorhagen** [comment]
> Thanks for taking this.
> 
> Two remarks:
> 
>     * Should we align the values with https://github.com/mendersoftware/meta-mender/pull/1416/files ?

Yes, I was expecting a comment here. I was planning on just making a new commit, depending on which one is merged first.

> 
>     * Good changelog message, but we should slightly extend it (or create a new one?) to say that we are setting them to 90/45 (or whatever we decide on) for `meta-mender-demo`.

Good point. Adding it to the `Changelog`.
> 
> 
> See comment also.

**Comment by oleorhagen** [comment]
Holding off the merge pending: https://github.com/mendersoftware/meta-mender/pull/1416

**Review by kacf — APPROVED** [review]
Conclusion after live chat: Accept this, but leave it undocumented and remove changelog. We prefer that people use the mechanism of adding a custom `mender.conf` to the mender-client recipe, instead of having a new variable for every setting. But since we cannot use this mechanism ourselves (it would conflict if the user also does it), we need these variables for our demo layer.

**Comment by lluiscampos** [comment]
Rebased on top of latest master.

@mender-test-bot start pipeline :baseball:

**Comment by lluiscampos** [comment]
@mender-test-bot start pipeline :rugby_football:

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**